tools/circleci_dependencies.sh

```diff
@@ -11,6 +11,7 @@ echo "Installing dependencies"
 python -m pip install --upgrade --progress-bar off \
     -r requirements.txt -r requirements_testing.txt -r requirements_doc.txt \
     "mne-qt-browser[opengl] @ git+https://github.com/mne-tools/mne-qt-browser.git@main" \
+    "mne-python @ git+https://github.com/larsoner/mne-python.git@feature/TD-nirs_snirf" \
```

Comment:
The forward slash is causing problems, let's try this
```suggestion
    "https://github.com/larsoner/mne-python/archive/refs/heads/feature/TD-nirs_snirf.zip" \
```